test/Numerics/DGMethods/compressible_navier_stokes_equations/three_dimensional/test_buoyancy.jl

```diff
@@ -101,6 +101,8 @@ ClimateMachine.init()
         for (key2, orientation) in orientations
             key = key1 * key2
 
+            println("running ", key)
```

Comment:
leftover debug commet

Reply:
this is actually so it's easy to check which state check test is which during the CI